### ISSUE TITLE
Transform label distribution from [-1,1] to [0,1] to run this code

### ISSUE THREAD

**Issue by JourneyGo** [crazydemo/facial-attribute-classification-with-graph#1]
This code compute loss using sparse style as below:

         cross_entropy_cln[i] = tf.reduce_mean(tf.nn.sparse_softmax_cross_entropy_with_logits(labels=y[:,i],logits=logits_cln[i]))

'labels' should be [0,1,...,class_num], or there will be Nan value everywhere when running on GPU.

**Comment by JourneyGo** [comment]
Not an issue, just for reminding.
